services/workflows-service/src/alert/alert.service.ts

```diff
@@ -16,6 +16,15 @@ import { AlertExecutionStatus } from './consts';
 
 // TODO: move to utils
 
+const findValNoCase = (obj: { [key: string]: unknown }, prop: string) => {
+  prop = (prop + '').toLowerCase();
+  for (let p in obj) {
+    if (p in obj && prop === `${p}`.toLowerCase()) {
```

Comment:
Use `.hasOwnProperty`
```suggestion
  for (const p in obj) {
    if (obj.hasOwnProperty(p) && prop === p.toLowerCase()) {
```
Or use `Object.keys()`
```suggestion
  for (const p of Object.keys(obj)) {
    if (prop === p.toLowerCase()) {
```

Reply:
property can be also non string 
that's why i've used `${p}`

Reply:
eslint rule
  7:13  error    Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins